### ISSUE TITLE
inkscape args option outdated?

### ISSUE THREAD

**Issue by paptigshiyf** [aminophen/chemobabel#2]
Praise your Project !!!!! Its hard to find another package that can transfer SMILES names to 2D structures in Latex !! 
It really helps!

BTW, It seems that in the newest vewsion of inkscape, `-f` option is no longer allowed, while `--export-pdf=${filename}` is also deprecated and not available. So I use `--export-type=pdf` instead in **`chemobabel.sty`, line 42**:

```latex
% \newcommand\chemob@bel@inkscapecmd[2]{inkscape -f #1.svg --export-#2=#1.#2}
\newcommand\chemob@bel@inkscapecmd[2]{inkscape #1.svg --export-type=#2}
```
And it works for me now on Win10 PC. 
Appreciate again!

**Comment by aminophen** [comment]
Thank you for the report; I've never used a new version of Inkscape. It seems that a major rewrite of code in version 1.0 changed the option names; I think I should add a version check of Inkscape to use correct options.

**Comment by aminophen** [comment]
Sorry for late, fixed in ea2016135acb10c308cdf788875a43573b74ae77

As a backward compatibility, if someone is using older inkscape, he/she can add `inkscape-old` package option.
